src/schema.sql

```diff
@@ -5,28 +5,44 @@ CREATE EXTENSION IF NOT EXISTS pgcrypto;
 CREATE EXTENSION IF NOT EXISTS hstore;
 
 DROP TABLE IF EXISTS webhooks;
+DROP TABLE IF EXISTS meta;
+DROP TABLE IF EXISTS users_tokens;
+DROP TABLE IF EXISTS bounds;
+DROP TABLE IF EXISTS tiles;
+DROP TABLE IF EXISTS geo;
+DROP TABLE IF EXISTS geo_history;
+DROP TABLE IF EXISTS styles;
+DROP TABLE IF EXISTS deltas;
+DROP TABLE IF EXISTS users;
+
+CREATE TABLE users (
+    id          BIGSERIAL PRIMARY KEY,
+    access      TEXT,
+    username    TEXT UNIQUE NOT NULL,
+    password    TEXT NOT NULL,
+    email       TEXT UNIQUE NOT NULL,
+    meta        JSONB
+);
```

Comment:
Same here, why is this table being moved? Can it stay in place?

Reply:
No, it needs to be moved to be created as other referenced tables now depend on it

Reply:
Removed in https://github.com/mapbox/Hecate/pull/206/commits/7493e0efafaeec0af7b70cbab56df3f1b16525d5 -- while these are changes we should make, i'm removing them from the current PR to reduce scope for what will already be a large change.